src/rct1.c

```diff
@@ -2048,7 +2078,92 @@ static void rct1_import_ride(rct1_s4 *s4, rct_ride *dst, rct1_ride *src)
 	memset(dst, 0, sizeof(rct_ride));
 
 	dst->type = RCT1RideTypeConversionTable[src->type];
-	dst->subtype = src->type;
+
+	// Make sure the right vehicle is chosen.
+	// https://github.com/OpenRCT2/OpenRCT2/wiki/RCT1-ride-and-vehicle-types-and-their-RCT2-equivalents
+	switch(src->vehicle_type)
+	{
+		case 3:
+			dst->subtype = 73;
+			break;
+		case 7:
+			dst->subtype = 85;
+			break;
+		case 9:
+			dst->subtype = 86;
+			break;
+		case 10:
+			dst->subtype = 87;
+			break;
+		case 11:
+			dst->subtype = 88;
+			break;
+		case 15:
+			dst->subtype = 89;
+			break;
+		case 17:
+			dst->subtype = 90;
+			break;
+		case 20:
+			dst->subtype = 91;
+			break;
+		case 21:
+			dst->subtype = 92;
+			break;
+		case 22:
+			dst->subtype = 93;
+			break;
+		case 24:
+			dst->subtype = 94;
+			break;
+		case 32:
+			dst->subtype = 95;
+			break;
+		case 33:
+			dst->subtype = 96;
+			break;
+		case 34:
+			dst->subtype = 97;
+			break;
+		case 49:
+			dst->subtype = 98;
+			break;
+		case 52:
+			dst->subtype = 99;
+			break;
+		case 58:
+			dst->subtype = 100;
+			break;
+		case 59:
+			dst->subtype = 101;
+			break;
+		case 60:
+			dst->subtype = 102;
+			break;
+		case 61:
+			dst->subtype = 103;
+			break;
+		case 63:
+			dst->subtype = 104;
+			break;
+		case 71:
+			dst->subtype = 105;
+			break;
+		case 73:
+			dst->subtype = 106;
+			break;
+		case 79:
+			dst->subtype = 107;
+			break;
+		case 82:
+			dst->subtype = 108;
+			break;
+		case 85:
+			dst->subtype = 109;
+			break;
+		default:
+			dst->subtype = src->type;
+	}
```

Comment:
Could you use an array / table instead of a switch?
Like RCT1DefaultObjectsRides and others are done.